### PR TITLE
CMRARC-505 - now checks http response code.

### PR DESCRIPTION
**Pull request by cgokey**

### CODE REVIEW

File: app/models/cmr.rb [hunk 1]
```diff
@@ -369,8 +369,10 @@ def self.get_raw_granule_results(concept_id)
   # returns true if a granule with the specified concept id exists in cmr
   def self.raw_granule?(concept_id)
     url = Cmr.api_url('granules', 'echo10', 'concept_id': concept_id)
-    granule_xml = Cmr.cmr_request(url).parsed_response
-    granule_results = convert_to_hash('echo10', granule_xml)['results']
+    response = Cmr.cmr_request(url)
+    return false if response.code != 200
```
Comment (charleshuang80) [inline app/models/cmr.rb]: would it be helpful to log anything here?

File: app/models/cmr.rb [hunk 2]
```diff
@@ -686,6 +688,7 @@ def self.get_cmr_base_url
     if cmr_base_url.nil?
       cmr_base_url = 'https://cmr.earthdata.nasa.gov'
     end
+    cmr_base_url = 'https://cmr.earthdata.nasa.gov'
```
Comment (charleshuang80) [inline app/models/cmr.rb]: this either seems redundant or that you can remove all the lines above it in the method.

File: app/models/cmr.rb (hunk 1 above)
Comment (cgokey) [reply app/models/cmr.rb]: I added a log message.

File: app/models/cmr.rb (hunk 2 above)
Comment (cgokey) [reply app/models/cmr.rb]: debugging accidently made it into the commit.  removed.

**Pull request merged**